package.json

```diff
@@ -48,7 +48,8 @@
   },
   "dependencies": {
     "prop-types": "^15.6.2",
-    "react-native-animatable": "1.3.3"
+    "react-native-animatable": "1.3.3",
+    "react-native-screens": "^1.0.0-alpha.23"
```

Comment:
🤔 wrong package?

Reply:
Followed the installation steps from react-navigation, which needs react-native-reanimated, react-native-screens and react-native-gesture-handler 😃

Reply:
I mean, wrong `package.json`?

Reply:
Dammit yes, it's fixed